pyproject.toml

```diff
@@ -37,4 +37,6 @@ six = "^1.15.0"
 netaddr = "^0.8"
 
 [tool.poetry.dev-dependencies]
-nose = "^0.10.0b1"
+nose = "^1.3"
```

Comment:
Please kill nosetests with fire. It is native python2 code and used 2to3 in setuptools to work on Python3, which setuptools drop in 58.0.